sdk/identity/azure-identity/test/live/main.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <gtest/gtest.h>
+
+int main(int argc, char** argv)
+{
+  testing::InitGoogleTest(&argc, argv);
```

Comment:
If this tests are going to run in Linux with libcurl, we need to add global_init and global_clean up here.
See the Azure core main.cpp

Reply:
BTW, it looks like it was running fine without that, but I'll add it.